api/utils/chainUtils.ts

```diff
@@ -0,0 +1,35 @@
+export function getChains() {
+    return {
+        "1": "ethereum",
+        "10": "optimism",
+        "25": "cronos",
+        "56": "bsc",
+        "137": "polygon",
+        "250": "fantom",
+        "288": "boba",
+        "1088": "metis",
+        "1284": "moonbeam",
+        "1285": "moonriver",
+        "8217": "klaytn",
+        "42161": "arbitrum",
+        "43114": "avalanche",
+        "53935": "dfk",
+        "121014925": "terra",
+        "1313161554": "aurora",
+        "1666600000": "harmony"
+    }
+}
+
+export function getChainIdNums() {
+    let res: number[] = []
+    for (let key of Object.keys(getChains())) {
+        res.push(parseInt(key))
+    }
+    return res
+}
+
+export function getChainNameFromId(chainId: number|string) {
+    let chainIdStr: string = chainId.toString()
+    // @ts-ignore
```

Comment:
This keeps throwing me an error. Is there a cleaner way ?

Reply:
The problem here is you're using an enum to query a dictionary, so if we get:

```typescript
undefined === getChainNameFromId("999")
undefined != string
```

This method doesn't actually return a string